### DISCUSSION TITLE
Jellyfin on Fedora with NVIDIA and Wayland

### DISCUSSION THREAD

**Discussion by gbrsni** [jellyfin/jellyfin-desktop#511] [General]
When on Fedora, with the proprietary NVIDIA drivers (535.113.01 as of writing) installed from rpmfusion and using Wayland, the Jellyfin Media Player flatpak would open to a black screen. Simply disabling "Wayland windowing system" (socket=wayland) in Flatseal fixed the issue and it now works flawlessly. This may be interesting to other Fedora users with NVIDIA cards. I wasn't sure if this warranted opening an issue.

**Comment by LegitGreenBoi** [comment]
Is this an issue for AMD cards as well, or just Nvidia?

It's probably worth opening this issue over at the Flathub repository for Jellyfin:
https://github.com/flathub/com.github.iwalton3.jellyfin-media-player/issues

**Comment by RigleGit** [comment]
This helped me today with Fedora 41. Thank you!

**Comment by cyfrit** [comment]
This also worked on Arch Linux, thank you! This should be a common problem, not just on Fedora
